lib/final-api/v1/http/ddtf_build.rb

```diff
@@ -121,6 +122,21 @@ def atom_response
           }
         end
 
+        def execution_logs
+          puts build.message.inspect
+
+          el = JSON.parse(build.message || [].to_json)
+          els = el.sort { |x,y| x['position'].to_i <=> y['position'].to_i }
+          messages = els.map do |x|
```

Comment:
I would rather see some better variable name than 'x', when it's not one line block.